amundsen_application/models/jira_issue.py

```diff
@@ -0,0 +1,15 @@
+from amundsen_application.base.base_issue import BaseIssue
+
+
+class JiraIssue(dict, BaseIssue):
```

Comment:
I wonder if we are making this a bit complicated by having `BaseIssue` as abstraction class. How about just having plain `Issue` class? Other implementation can add optional fields later on..

Reply:
yeah i think you're right. i'll remove it.